src/main/java/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package service.impl;
+
+import db.Storage;
+import java.util.List;
+import model.FruitTransaction;
+import service.ShopService;
+import strategy.OperationStrategy;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
The class should not start with an empty line. Please remove the empty line at the beginning of the class declaration.